src/components/alert/__snapshots__/__spec__.js.snap

```diff
@@ -1,81 +1,185 @@
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </Heading>
+              </div>
+              <div
+                className="carbon-dialog__content"
+              >
+                <div
+                  className="carbon-dialog__inner-content"
+                  style={Object {}}
+                />
+              </div>
+              <Icon
+                bgSize="small"
+                className="carbon-dialog__close"
+                data-element="close"
+                onBlur={[Function]}
+                onClick={[Function]}
+                tabIndex="0"
+                type="close"
+              >
+                <span
+                  className="carbon-icon carbon-dialog__close icon-close"
+                  data-component="icon"
+                  data-element="close"
+                  onBlur={[Function]}
+                  onClick={[Function]}
+                  tabIndex="0"
+                  type="close"
+                />
+              </Icon>
+            </div>
+          </CSSTransitionGroupChild>
+        </div>
+      </TransitionGroup>
+    </CSSTransitionGroup>
+    <CSSTransitionGroup
+      component="div"
+      transitionAppear={false}
+      transitionEnter={true}
+      transitionEnterTimeout={500}
+      transitionLeave={true}
+      transitionLeaveTimeout={500}
+      transitionName="modal-background"
+    >
+      <TransitionGroup
+        childFactory={[Function]}
+        component="div"
+        transitionAppear={false}
+        transitionEnter={true}
+        transitionEnterTimeout={500}
+        transitionLeave={true}
+        transitionLeaveTimeout={500}
+        transitionName="modal-background"
       >
-        <div
-          className="carbon-dialog__inner-content"
-          style={Object {}}
-        />
-      </div>
-      <Icon
-        bgSize="small"
-        className="carbon-dialog__close"
-        data-element="close"
-        onBlur={[Function]}
-        onClick={[Function]}
-        tabIndex="0"
-        type="close"
-      />
-    </div>
-  </CSSTransitionGroup>
-  <CSSTransitionGroup
-    component="div"
-    transitionAppear={false}
-    transitionEnter={true}
-    transitionEnterTimeout={500}
-    transitionLeave={true}
-    transitionLeaveTimeout={500}
-    transitionName="modal-background"
-  >
-    <div
-      className="carbon-modal__background"
-    />
-  </CSSTransitionGroup>
-</div>
+        <div>
+          <CSSTransitionGroupChild
+            appear={false}
+            enter={true}
+            enterTimeout={500}
+            key=".0"
+            leave={true}
+            leaveTimeout={500}
+            name="modal-background"
+          >
+            <div
+              className="carbon-modal__background"
+              key=".0"
+            />
+          </CSSTransitionGroupChild>
+        </div>
+      </TransitionGroup>
+    </CSSTransitionGroup>
+  </div>
+</Alert>
```

Comment:
should avoid snapshotting mounted components, as it will create HTML tree of every single child component, whereas really all we care about is the top level (via shallow).

If we snapshot mounted components the snapshot will be incredibly brittle to changes in other components.